accessibility-checker.php

```diff
@@ -782,6 +783,27 @@ function edac_register_rules() {
 	}
 }
 
+/**
+ * Code that needs to run before the page is rendered
+ *
+ * @return void
+ */
+function edac_before_page_render() {
+	
+	// If the admin_bar is visible and we're not editing the post 
+	// assume we are on the frontend and the frontend highlighter is loading. 
+	global $pagenow;
+	if ( is_admin_bar_showing() && 'post.php' != $pagenow ) {
+
+		// Check the page if it hasn't already been checked.
+		global $post;
+		$checked = get_post_meta( $post->ID, '_edac_post_checked', true );
+		if ( false === boolval( $checked ) ) {
```

Comment:
We need to check if the current post type is set to be scanned in the plugin settings.

Reply:
I think edac_validate handles that as a pre-check:
```
function edac_validate( $post_ID, $post, $action ) {
        // check post type.
        $post_types = get_option( 'edac_post_types' );
        if ( is_array( $post_types ) && ! in_array( $post->post_type, $post_types, true ) ) {
                return;
        }
```